husky_components/datagrid/decorators/table-view.js

```diff
@@ -1060,15 +1060,21 @@ define(function() {
          * @param attribute {String}
          */
         showInput: function(recordId, attribute) {
-            var $cell, $inputs;
+            var $cell, $inputs, $inputField, $inputWrapper;
+
             if (!attribute) {
                 $inputs = this.sandbox.dom.find('.' + constants.editableInputClass, this.table.rows[recordId].$el);
                 attribute = this.sandbox.dom.data(this.sandbox.dom.parents($inputs[0], 'td'), 'attribute');
             }
+
             $cell = this.table.rows[recordId].cells[attribute].$el;
-            this.sandbox.dom.show(this.sandbox.dom.find('.' + constants.inputWrapperClass, $cell));
-            this.sandbox.dom.focus(this.sandbox.dom.find('.' + constants.editableInputClass, $cell));
-            this.sandbox.dom.select(this.sandbox.dom.find('.' + constants.editableInputClass, $cell));
+            $inputField = this.sandbox.dom.find('.' + constants.editableInputClass, $cell);
+            $inputWrapper = this.sandbox.dom.find('.' + constants.inputWrapperClass, $cell);
+            this.sandbox.dom.data($inputField, 'edit', 'on');
```

Comment:
@wachterjohannes Wouldn't it be a good idea to just remove the enter handler? Would solve the issue with having multiple requests in a better way.